api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/EmailPayloadFactory.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.dhsc.htbhf.claimant.communications;
+
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PaymentCycleVoucherEntitlement;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.message.payload.EmailMessagePayload;
+import uk.gov.dhsc.htbhf.claimant.message.payload.EmailType;
+
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.dhsc.htbhf.claimant.message.EmailTemplateKey.*;
+import static uk.gov.dhsc.htbhf.claimant.message.MessagePayloadFactory.formatPaymentAmountSummary;
+import static uk.gov.dhsc.htbhf.claimant.message.MoneyUtils.convertPenceToPounds;
+
+/**
+ * Builds the message payload required to send an email message. The email template has parameterised values
+ * which are contained in the emailPersonalisation Map. All monetary amounts are formatted into pounds and the breakdown
+ * of voucher payments has been detailed in a bullet point list - any vouchers which are missing are replaced with a
+ * single blank line so that we don't have any empty bullet point in the email.
+ */
+@Component
+public class EmailPayloadFactory {
```

Comment:
I think there is a bit of inconsistency of naming between `EmailMessagePayload` and `EmailPayloadFactory`

Reply:
Renamed to EmailMessagePayloadFactory